scripts/ocsp-stapling.test

```diff
@@ -234,8 +234,12 @@ if [ ! -f "$ready_file" ]; then
     exit 1
 else
     # should fail if ocspstapling is also enabled
-    echo "hi" | openssl s_client -status $V4V6_FLAG -connect ${LOCALHOST}:$port1 -cert ./certs/client-cert.pem -key ./certs/client-key.pem -CAfile ./certs/ocsp/root-ca-cert.pem 2>&1 | tee /dev/stderr | fgrep -q 'self signed certificate in certificate chain'
-    if [ $? -neq 0 ]; then
+    OPENSSL_OUTPUT=$(echo "hi" | openssl s_client -status $V4V6_FLAG -connect ${LOCALHOST}:$port1 -cert ./certs/client-cert.pem -key ./certs/client-key.pem -CAfile ./certs/ocsp/root-ca-cert.pem 2>&1)
+    OPENSSL_RESULT=$?
+    echo "$OPENSSL_OUTPUT"
+    fgrep -q 'self signed certificate in certificate chain' <<< "$OPENSSL_OUTPUT"
+    FGREP_RESULT=$?
+    if [ $OPENSSL_RESULT -eq 0 -a FGREP_RESULT -ne 0 ]; then
```

Comment:
whoops!  you did `FGREP_RESULT -ne 0` in the test, instead of `$FGREP_RESULT -ne 0`.

Reply:
Very good catch. Fixed and pushed.